packages/arb-token-bridge-ui/src/hooks/__tests__/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { RenderHookResult, act, renderHook } from '@testing-library/react'
```

Comment:
The name is a bit misleading, I'd expect this to be reusable for any hooks we have, but currently, it's tied to `useBalance` only

Reply:
Also, IMO it should not be part of hooks folder, it's an utility for unit test only, not for hooks.

Reply:
hmm it's under `__tests__/` though

Reply:
I forgot we have no dedicated test folder on the bridge, make sense then